web/src/containers/admin/MyAccount.js

```diff
@@ -110,12 +117,37 @@ class MyAccount extends Component {
             value={position || ''}
             onChange={this.handleEditAccount}
           />
-          <Password
-            className="mb2"
-            title="Change password"
-            value={password}
-            onChange={this.handleEditAccount}
-          />
+          <div className="ds-l-row ds-u-padding-x--2">
+            <TextField
+              label="Password"
+              ariaLabel="Current password"
```

Comment:
Since this doesn't read the current password aloud, can this say something like "Current password obscured for security" or somesuch? Otherwise I fear this might read out, "Current password: Asterisk Asterisk Asterisk"

Reply:
Because the current password text field is disabled, the screen reader will skip it altogether and go straight to the "change password" button.  So that part seems okay, but the button text is the same no matter what state the form is in.  The only thing that changes is the lock/unlock icon.  So in bde586d, I added an ARIA label to the button.